website/content/en/docs/migration/v0.19.0.md

```diff
@@ -46,3 +46,5 @@ However, any script or code that is depending on this condition reason must be u
 to use `UpgradeError` instead of `UpdateError`.
 
 _See [#3269](https://github.com/operator-framework/operator-sdk/pull/3269) for more details._
+
+[migration-guide]: docs/golang/project_migration_guide
```

Comment:
```suggestion
[migration-guide]: /docs/golang/project_migration_guide
```

I think?

Reply:
I will use the full with the tag to be sure. https://v0-19-x.sdk.operatorframework.io/docs/golang/project_migration_guide/